packages/flutter_localizations/lib/src/l10n/date_localizations.dart

```diff
@@ -25,7 +25,7 @@ const Map<String, dynamic> dateSymbols = <String, dynamic>{
       r'''س''',
       r'''ك''',
       r'''ب''',
-      r'''د''',
+      r'''د'''
```

Comment:
Is the removal of trailing commas intentional (did the generator script change to cause this?)?

Reply:
It's not intentional and gen_localizations.dart didn't change in a way that would cause this. The app uses dartfmt which may have changed.